File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// YugiLimitRegulation SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class YugiLimitRegulationMakeContext
{
    public static function call(array $ctxmap, ?YugiLimitRegulationContext $basectx): YugiLimitRegulationContext
    {
        return new YugiLimitRegulationContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# YugiLimitRegulation SDK utility: make_context
require_relative '../core/context'
module YugiLimitRegulationUtilities
  MakeContext = ->(ctxmap, basectx) {
    YugiLimitRegulationContext.new(ctxmap, basectx)
  }
end
